scripts/check-locales.sh

```diff
@@ -1,8 +1,20 @@
 #! /bin/bash
 
-errormessage=`cmp <(jq 'path(..)|map(tostring)|join(".")' src/locales/en.json) <(jq 'path(..)|map(select(length > 0)|tostring)|join(".")' src/locales/fr.json) 2>&1`
+en_paths=$(mktemp "/tmp/check-locales-XXX")
```

Comment:
Wouldn't it make sense to move this script in a shared repo? Maybe in cozy-ui/i18n?

Reply:
I am planning to do this in cozy-scripts.